README.md

```diff
@@ -1 +1,27 @@
-# magpies-apple-clone
\ No newline at end of file
+# HTML-Forms
+This project consists of cloning the Apple webpage using HTML and CSS.
+
+# Built With
+- HTML
+- CSS
+
+Link to live page
+https://raw.githack.com/djallas/magpies-apple-clone/master/index.html
```

Comment:
Please double-check this link, it is not working. You should paste the URL to the `index.html` of your desired branch into the https://raw.githack.com/ input. The URL you should paste there for this project is https://github.com/djallas/magpies-apple-clone/blob/develop/index.html, then you can share the URL give under `Use this URL for development`: https://raw.githack.com/djallas/magpies-apple-clone/develop/index.html.